src/libraries/Microsoft.Extensions.Http/src/Logging/LoggingScopeHttpMessageHandler.cs

```diff
@@ -113,12 +113,12 @@ public static class EventIds
 
             public static IDisposable? BeginRequestPipelineScope(ILogger logger, HttpRequestMessage request)
             {
-                return _beginRequestPipelineScope(logger, request.Method, GetUriString(request.RequestUri));
+                return _beginRequestPipelineScope(logger, request.Method, LoggingHttpMessageHandler.Log.GetUriString(request.RequestUri));
```

Comment:
Let's not use one handler from inside of the unrelated other? Given the (size of the) change @MihaZupan suggested, I think we might need a separate `LogHelper`/`RedactionHelper` class or something